src/app.tsx

```diff
@@ -1,50 +1,71 @@
-import React, {useEffect, useState} from 'react';
-import { createRoot } from 'react-dom/client';
-import './app.css';
+import React, { useEffect, useState } from "react";
+import { createRoot } from "react-dom/client";
+import "./app.css";
+
+import { HashRouter as Router, Link, Route, Routes } from "react-router-dom";
+import Settings from "./pages/settings/settings";
 
 const CountdownTimer = () => {
```

Comment:
**Actionable**
Please remove the CountdownTimer. It was just a placeholder and its not needed anymore.